dist/stat_data.py

```diff
@@ -680,6 +680,7 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     TxnStat('txn_begin', 'transaction begins'),
     TxnStat('txn_commit', 'transactions committed'),
+    TxnStat('txn_ckpt_tree_duration', 'transaction checkpoint tree file duration (usecs)'),
```

Comment:
Two things - (1) this stat description shouldn't have the word tree, i.e. it should be `transaction checkpoint tree duration (usecs)`. And (2), `txn_hs_ckpt_duration` is in the wrong spot - would you please move both this new stat and `txn_hs_ckpt_duration` up to the `CheckpointStat` section of this file?

Reply:
Sorry - it needs to be a plural too. Since this is the time spent on *all* dhandles. So it should be `transaction checkpoint trees duration (usecs)`.

Reply:
hi, @korteland 
Thanks for your direction, I have change the code, please review the code again.

thanks.